### ISSUE TITLE
Use fuzzysort for username suggestions

### ISSUE THREAD

**Issue by andrewda** [gitpoint/git-point#274]
It's got a nicer API than whatever I spent 3 minutes searching for before.

```javascript
import fuzzysort from 'fuzzysort';

// configure fuzzysort
fuzzysort.highlightMatches = false;
fuzzysort.threshold = 500; // will need some testing

// ...

const results = fuzzysort.go(inputUsername, arrayOfUsernames);
```

Something like that should work 😄

**Comment by housseindjirdeh** [comment]
All for a cleaner API :) Thank you @andrewda

**Comment by housseindjirdeh** [comment]
Just to add some clarity for those new to the repo, we're currently using `fuzzy-search` to show other users that can be tagged when typing an issue comment (auto-completion):

![mentions-area](https://user-images.githubusercontent.com/12476932/28508524-8c4a3f6e-7009-11e7-8d23-be9d8d2f44b0.gif)

Will be nice if we can swap the library we're using to use `fuzzysort` since it seems to have a cleaner API :)

Please don't hesitate to ask questions at anytime 💃

**Comment by alejandronanez** [comment]
I'll take this one!

**Comment by chinesedfan** [comment]
@alejandronanez How about your progress? I want to also take a look.

**Comment by alejandronanez** [comment]
Hey @chinesedfan I'm working on it.

**Comment by chinesedfan** [comment]
Closed by #473.
